cron/builder/main.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	codeship "github.com/codeship/codeship-go"
+)
+
+// EnvKeyCSOrganization is the environment variable for
+// the Codeship organization
+const EnvKeyCSOrganization = "CS_ORGANIZATION"
+
+// EnvKeyCSPassword is the environment variable for
+// the Codeship password
+const EnvKeyCSPassword = "CS_PASSWORD"
+
+// EnvKeyCSUsername is the environment variable for
+// the Codeship username
+const EnvKeyCSUsername = "CS_USERNAME"
+
+// EnvKeyCSProjectUUID is the environment variable for
+// the uuid of the target Codeship project
+const EnvKeyCSProjectUUID = "CS_PROJECT_UUID"
+
+// EnvKeyCSBuildReference is the environment variable for
+// the desired reference for the build of the Codeship project
+// e.g. "20.04"  (which will have "heads/" tacked onto the beginning, if it isn't there)
+const EnvKeyCSBuildReference = "CS_BUILD_REFERENCE"
+
+func getRequiredString(envKey string, configEntry *string) error {
+	if *configEntry != "" {
+		return nil
+	}
+
+	value := os.Getenv(envKey)
+	if value == "" {
+		return fmt.Errorf("required value missing for environment variable %s", envKey)
+	}
+	*configEntry = value
+
+	return nil
+}
+
+func (c *BuilderConfig) init() error {
```

Comment:
I don't see where `init` is called. Should it be?

Reply:
Oops. Right. It's just a struct method not a package level function.